plugins/modem-manager/modem-manager.quirk

```diff
@@ -78,3 +78,16 @@ ModemManagerFirehoseProgFile = prog_firehose_sdx6x.elf
 [PCI\VID_1EAC&PID_1007]
 Summary = Quectel RM520 firehose prog file
 ModemManagerFirehoseProgFile = prog_firehose_sdx6x.elf
+
+# Fibocom FM101
+[USB\VID_2CB7&PID_01A2]
+Summary = Fibocom Fibocom FM101-GL Module
+Flags = uninhibit-modemmanager-after-fastboot-reboot
+Flags = detach-at-fastboot-has-no-response
```

Comment:
does this work? I'd expected `Flags = uninhibit-modemmanager-after-fastboot-reboot,detach-at-fastboot-has-no-response`

Reply:
It works. It has been modified in one line.

Thanks.